### ISSUE TITLE
training generated images that appear pixelated or mosaic-like.

### ISSUE THREAD

**Issue by wsanm** [DepthAnything/Depth-Anything-V2#232]
Hello,
I have been attempting to train a model using the code you provided, during which I disabled the multi-GPU training functionality. Throughout the training process, I observed that the loss was decreasing as expected. However, other metrics started displaying NaN values. Additionally, the model generated images that appear pixelated or mosaic-like.
Could you please provide guidance on how to resolve these issues?
Thank you for your assistance.

**Comment by wsanm** [comment]
![QQMail_1734915754932](https://github.com/user-attachments/assets/fe918245-5cca-47c6-8dba-eeb190f86105)

**Comment by hcscysh** [comment]
@wsanm @wangyue7777. I have the same problem,have you found a solution?

**Comment by tomyslavs** [comment]
Same mosaic depth
![Image](https://github.com/user-attachments/assets/0ba9155e-54da-4610-9550-5a588a8ee5d3)

**Comment by Albertchamberlain** [comment]
> Same mosaic depth  相同的马赛克深度 ![Image](https://github.com/user-attachments/assets/0ba9155e-54da-4610-9550-5a588a8ee5d3)

same problem

**Comment by ChickenSellerRED** [comment]
![Image](https://github.com/user-attachments/assets/2dd0e260-cee5-4344-8120-8cf5e5ef7c7f)

Same here. I got this output after finetune in hypersim. Has anyone finds out the reason?

**Comment by ChickenSellerRED** [comment]
I found out what happened. change this part in metric/train.py:
`    if args.pretrained_from:
        model.load_state_dict({k: v for k, v in torch.load(args.pretrained_from, map_location='cpu').items() if 'pretrained' in k}, strict=False)`

to

`    if args.pretrained_from:
        model.load_state_dict({k: v for k, v in torch.load(args.pretrained_from, map_location='cpu').items()}, strict=False)`

Result shows mosaic because I just loaded 'pretrained' part parameters, but other parameters(decoder part) are left random.
